internal/release/validate/validate.go

```diff
@@ -0,0 +1,133 @@
+package validate
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strings"
+
+	"cuelang.org/go/cue"
+	"cuelang.org/go/cue/cuecontext"
+	cueerrors "cuelang.org/go/cue/errors"
+	"cuelang.org/go/encoding/gocode/gocodec"
+
+	"github.com/davidmdm/x/xerr"
+
+	"github.com/nestoca/joy/api/v1alpha1"
+	"github.com/nestoca/joy/internal"
+	"github.com/nestoca/joy/internal/config"
+	"github.com/nestoca/joy/internal/helm"
+	"github.com/nestoca/joy/internal/release/render"
+)
+
+type ValidateParams struct {
```

Comment:
Let's try to stay consistent:
```suggestion
type ValidateOpts struct {
```